pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -588,6 +590,7 @@ func (deploy *NewDeploy) updateFunction(ctx context.Context, oldFn *fv1.Function
 	if oldFn.Spec.Environment != newFn.Spec.Environment ||
 		oldFn.Spec.Package.PackageRef != newFn.Spec.Package.PackageRef ||
 		oldFn.Spec.Package.FunctionName != newFn.Spec.Package.FunctionName {
+		deploy.logger.Info("deployment changed", zap.String("msg", "deployment changed"))
```

Comment:
Please remove this log statement or make it debug.

Reply:
done